webcompat/static/js/lib/models/comment.js

```diff
@@ -23,14 +12,14 @@ issues.Comment = Backbone.Model.extend({
   parse: function(response, jqXHR) {
     this.set({
       avatarUrl: response.user.avatar_url,
-      body: md.render(response.body),
+      body: response.body_html,
       commenter: response.user.login,
       commentLinkId: "issuecomment-" + response.id,
       createdAt:
         moment(Date.now()).diff(response.created_at, "weeks") > 1
           ? moment(response.created_at).format("YYYY-MM-DD")
           : moment(response.created_at).fromNow(),
-      rawBody: response.body
+      rawBody: response.body_html
```

Comment:
do we still need `rawBody`?
Because now we have two keys carrying the same content.
better to call now body, anywhere else in the code.

Reply:
You're right, they're the same. I'll remove the rawBody thing.
Thanks for noticing that Karl :)